src/ngx_http_lua_module.c

```diff
@@ -638,7 +639,8 @@ ngx_http_lua_init(ngx_conf_t *cf)
     volatile ngx_cycle_t       *saved_cycle;
     ngx_http_core_main_conf_t  *cmcf;
     ngx_http_lua_main_conf_t   *lmcf;
-#ifndef NGX_LUA_NO_FFI_API
+#if (!defined(NGX_LUA_NO_FFI_API)) \
+    || (defined(nginx_version) && nginx_version >= 1011011)
```

Comment:
I don't think we need the `defined(nginx_version)` part since this module explicitly states in the documentation that at least nginx 1.6.0 is needed where this `nginx_version` macro is already introduced. Please fix other similar places as well. Thank you!